fw/http_frame.c

```diff
@@ -804,6 +804,30 @@ tfw_h2_stream_id(TfwHttpReq *req)
 	return id;
 }
 
+int
+tfw_h2_stream_init_for_xmit(TfwHttpReq *req, unsigned long h_len,
```

Comment:
This function must be in `http_stream.c`. Also I see `tfw_h2_stream_create()`, `tfw_h2_stream_clean()`, `__tfw_h2_stream_del_from_queue()` and other `tfw_h2_stream_`-prefixed functions - let's move them to `http_stream.c`, which not only responsible for streams, but also is much smaller than `http_frame.c`.

Reply:
Yes, I think to do it in patch for 1196, but you right it's better to do it here